### DISCUSSION TITLE
Can you detach from and re-attach to Zellij sessions?

### DISCUSSION THREAD

**Discussion by eschnett** [zellij-org/zellij#365] [Q&A] [answered]
Since you compare Zellij to screen and tmux, I wonder whether you can detach from and re-attach to Zellij sessions running on a remote system. I didn't see any command related to session management in the documentation.

**Comment by auronandace** [comment, marked as answer]
Not yet. It is still being worked on: See issue #69

**Reply by auronandace** [reply]
It has been added since 0.12.0

**Reply by msharran** [reply]
How to detach from a session using `CLI` ?

**Reply by misterbracket** [reply]
Also interested in this 🙏 it would be lovely

**Reply by auronandace** [reply]
Not sure I understand the question. If you want to detach then that means you are already attached. Why would you need to send a CLI command? Is there a reason you can't detach using the built-in keybindings or session manager?

If you want to stop a running session without attaching you can use the following command:
```zellij kill-session name-of-sesion```

**Reply by misterbracket** [reply]
It would be lovely if the session manager capabilities would be available of CLI.  
In my use case, I would love to be able to run a custom bash script that detaches the current zellij session and opens a new session in a place I want. 

More specifically, I have a list  of directories that I want to jump to easily  and create a zellij session if one does not already exists.  
[This would be the equivalent](https://github.com/ThePrimeagen/.dotfiles/blob/master/bin/.local/scripts/tmux-sessionizer#L16-L25) of what I am trying to do with TMUX

Thanks for getting back to us on this 🥇

**Reply by auronandace** [reply]
Looks like there is an existing issue for this: #1963

**Comment by danielb2** [comment]
it's possible now ;) 
<kbd>ctrl + o d<kbd>
